### PR TITLE
added albumentation augmented features

### PR DESCRIPTION
**Pull request by jim-gyas**
added albumentation augmented features but not completely done with the testing part

### CODE REVIEW

File: src/SynthImage/line_extraction.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def get_max_width(self, lines, blank_img, font):
             # Draw the line on the blank image to calculate the bounding box
             temp_img = blank_img.copy()
             temp_draw = ImageDraw.Draw(temp_img)
-            line_bbox = temp_draw.textbbox((20, y), line, font=font)
+            line_bbox = temp_draw.textbbox((10, y), line, font=font)
```
Comment (kaldan007) [inline src/SynthImage/line_extraction.py:55]: Hard coded constant is not good practice. Please initialise a constant and use it. The best practice would be creating a config.py in the pkg and intialise the constant over there and import it to respective module where need.

File: src/SynthImage/rusting_augmentation.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        Returns:
+            PIL.Image.Image: Augmented image with rusting effect.
+        """
+        # Combine various transformations to simulate rust
+        aug = A.Compose(
+            [
+                A.OneOf(
+                    [
+                        A.GaussNoise(
+                            var_limit=(10.0, 50.0), p=0.5
```
Comment (kaldan007) [inline src/SynthImage/rusting_augmentation.py:28]: these numbers needs to be parameterise for better readability as well as gives u options to generate images.

**Review by kaldan007 — CHANGES_REQUESTED** [review]

File: src/SynthImage/line_extraction.py (hunk 1 above)
Comment (jim-gyas) [reply src/SynthImage/line_extraction.py:55]: ok sure, i will do that @kaldan007

**Comment by jim-gyas** [comment]
Hi @kaldan007, I have completed the changes based on your review. Could you please review the updated code to ensure everything is correct?

**Pull request merged**